wiki/Tournaments/Template/en.md

```diff
@@ -0,0 +1,146 @@
+---
+tags:
+- {ABBR}
+---
+
+<!--
+Find and replace those tags listed below with proper information, then delete this comment.
+{TOURNAMENT_NAME} = tournament name
+{MODE} = game mode
+{ABBR} = abbreviation of the tournament name followed by its iteration/year occurence i.e. MWC 4K 2018
+{ORGANISER} = organiser of the tournament
+{NTH} = installment ordinal number
+{TOPIC_NUM} = forum thread number
+{STATS_LINK} = link to overall statistics sheet (it should be a Google Drive link)
+You will still need to fill other items such as tables, etc.
+Use .png as a default format for images, since .jpg can't handle transparency.
+Alongside this comment, remember you will need to delete all comments across the article.
+-->
+
+# {TOURNAMENT_NAME}
+
+![{ABBR} Logo](img/logo.png)
+
+The **{TOURNAMENT_NAME}** (***{ABBR}***) is a country-based {MODE} tournament hosted by the {ORGANISER}. It is the {NTH} installment of the {TOURNAMENT_NAME}.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-01-01/2018-01-28 |
+| Live Drawings | 2018-01-29 12:00:00 UTC |
```

Comment:
sure we are going with `12:00:00 UTC`?

Reply:
Or maybe i should go with placeholder?
Since it's up to the organiser

Reply:
I think we were talking about the format here if it should have the minutes and no parenthesis.

As with the random numbers, it should be a placeholder. Like `YYYY-MM-DD HR:MIN:SEC`?

Also idk why seconds were included.

Reply:
juan used the `YYYY-MM-DD HR:MIN:SEC` for recent 4K WC 2018.
But i don't know, which is more suitable in this case. Should i remove `SEC`?